### ISSUE TITLE
Cant start on ubuntu 18 ec2 instance

### ISSUE THREAD

**Issue by ivanthemeh** [moovweb/react-storefront#37]
I am getting this error when running npm start on ubuntu 18.04 ec2 free tier 
```xdg-open: no method available for opening 'http://localhost:8080'```

any help please.. been at it for a bit now and nothing is working.

**Comment by dijs** [comment]
Are you trying to run this in development mode?

**Comment by ivanthemeh** [comment]
@dijs I am just trying to get it deployed on an ec2 for a prototype to show some folks.

**Comment by ivanthemeh** [comment]
I think I have narrowed it down to the webpack browser trying to open a window... I am having a tough time figuring out how to stop that from happening.

**Comment by dijs** [comment]
Yeah, the dev webpack build uses the `OpenBrowserPlugin`. You could just comment that out for now.

https://github.com/moovweb/react-storefront/blob/27b678ec6f510cee95ae0376d5e105594b861372/packages/react-storefront/webpack/client.js#L76

**Comment by markbrocato** [comment]
@dijs This should fix it. https://github.com/moovweb/react-storefront/pull/39/files

**Comment by dijs** [comment]
@markbrocato Nice!
